core-models/src/main/scala/com/mesosphere/usi/core/models/Ids.scala

```diff
@@ -0,0 +1,7 @@
+package com.mesosphere.usi.core.models
+
+
+case class PodId(value: String)
+case class ReservationId(value: String)
+case class AgentId(value: String)
+case class DeploymentId(value: String)
```

Comment:
Will there be any notions of deployments in the USI? I though that it not a case...